test/autorest/urlgroup/paths_test.go

```diff
@@ -0,0 +1,275 @@
+}
+
+func TestPathsFloatScientificNegative(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.FloatScientificNegative(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsFloatScientificPositive(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.FloatScientificPositive(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsGetBooleanFalse(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.GetBooleanFalse(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsGetBooleanTrue(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.GetBooleanTrue(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsGetIntNegativeOneMillion(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.GetIntNegativeOneMillion(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsGetIntOneMillion(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.GetIntOneMillion(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsGetNegativeTenBillion(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.GetNegativeTenBillion(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsGetTenBillion(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.GetTenBillion(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsStringEmpty(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.StringEmpty(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsStringNull(t *testing.T) {
+	client := getPathsOperations(t)
+	var s string
+	_, err := client.StringNull(context.Background(), s)
+	if err == nil {
+		t.Fatalf("Expected an error but did not receive one")
+	}
+}
+
+func TestPathsStringURLEncoded(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.StringURLEncoded(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+func TestPathsStringURLNonEncoded(t *testing.T) {
+	client := getPathsOperations(t)
+	result, err := client.StringURLNonEncoded(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+	helpers.VerifyStatusCode(t, result, http.StatusOK)
+}
+
+// TODO verify how to work with this type of string
```

Comment:
I would uncomment these out and use `t.Skip` so that we can grep for all disabled tests.

Reply:
Done